rust-runtime/aws-smithy-http/Cargo.toml

```diff
@@ -11,8 +11,9 @@ license = "Apache-2.0"
 repository = "https://github.com/awslabs/smithy-rs"
 
 [features]
-rt-tokio = ["dep:tokio-util", "dep:tokio", "tokio?/rt", "tokio?/fs", "tokio?/io-util", "tokio-util?/io"]
-event-stream = ["aws-smithy-eventstream"]
+hyper014 = ["dep:hyper"]
+event-stream = ["aws-smithy-eventstream", 'hyper014']
```

Comment:
I don't think event stream actually uses hyper, but it's pulling in `http-body`'s `Body` trait via the hyper crate. If you fix that import, you can get rid of it pulling in the hyper feature.

Reply:
Addressed in c1cbe28.